OpenRA.Mods.Common/Traits/SupportPowers/NukePower.cs

```diff
@@ -22,10 +22,15 @@ class NukePowerInfo : SupportPowerInfo, IRulesetLoaded, Requires<BodyOrientation
 	{
 		[WeaponReference]
 		[Desc("Weapon to use for the impact.",
-			"But also image to use for the missile.",
-			"Requires an 'up' and a 'down' sequence on the image.")]
+			"Also image to use for the missile.")]
 		public readonly string MissileWeapon = "";
```

Comment:
Based on the description and the blank default I assume this wants a `[FieldLoader.Require]` attribute.